<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SortingWindow</class>
 <widget class="QWidget" name="SortingWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>409</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SortingWindow</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="min_value_label">
       <property name="text">
        <string>Min Value</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="min_value_input"/>
     </item>
     <item>
      <widget class="QLabel" name="max_value_label">
       <property name="text">
        <string>Max Value</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="max_value_input"/>
     </item>
     <item>
      <widget class="QLabel" name="length_label">
       <property name="text">
        <string>Length</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="length_input"/>
     </item>
     <item>
      <widget class="QPushButton" name="generate">
       <property name="text">
        <string>Generate</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableWidget" name="array">
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>45</number>
     </attribute>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QComboBox" name="sort_selection">
       <item>
        <property name="text">
         <string>------</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Bubble sort</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Selection sort</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Quick sort</string>
        </property>
       </item>

      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="sort">
       <property name="text">
        <string>Start sorting</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
